arithmoi.cabal

```diff
@@ -1,5 +1,5 @@
 name                : arithmoi
-version             : 0.4.1.3
+version             : 0.4.2.0
 cabal-version       : >= 1.10
 author              : Daniel Fischer
```

Comment:
should we drop the -fllvm flag? I'm pretty sure motivated users can figure out --ghc-options="-fllvm" themselves! (do we have benchmarks to validate exposing it at our level at all?)